packages/website/src/app.jsx

```diff
@@ -122,12 +113,41 @@ const App = () => {
                     <Route path='/shop/:slug' component={Item} />
                     <Route path='/basket' component={Basket} />
                     <Route exact path='/checkout' component={Checkout} />
-                    <Route exact path='/checkout/success' component={CheckoutSuccess} />
+                    <Route
+                      exact
+                      path='/checkout/success'
+                      component={CheckoutSuccess}
+                    />
                   </>
                 </Flag>
 
-                <Route path='/track-trace' component={() => window.location.replace('https://docs.google.com/forms/d/e/1FAIpQLSc9AFF9oQI2bDKWf9heplIi3hPQE6Ok2v-zGObByD9T0EjlbQ/viewform')} />
+                <Route
+                  path='/track-trace'
+                  component={() =>
+                    window.location.replace(
+                      'https://docs.google.com/forms/d/e/1FAIpQLSc9AFF9oQI2bDKWf9heplIi3hPQE6Ok2v-zGObByD9T0EjlbQ/viewform'
+                    )}
+                />
 
+                <Switch>
+                  {/* static pages */}
+                  {/* <Route exact path="/" component={Dynamic} /> */}
+                  {/* <Route path="/club" component={Dynamic} /> */}
+                  {/* <Route path="/ethos" component={Ethos} /> */}
+                  {/* <Route path="/diversity" component={Diversity} /> */}
```

Comment:
I can clean these away as not needed actually

Reply:
Please do, we can bring them back via the git history if necessary.

We should also be able to delete the content pages these routes linked to as well now, right?

Reply:
yep will clean these and delete the files too when doing the PR updates👍